src/scripts/tab.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+let tabContent = function () {
+  let lists = document.querySelectorAll('.accordion a');
+  let rowItems = document.getElementsByClassName('items_active');
+  let numberOfRowItems = rowItems.length;
+
+  let listClickBack = function (item) {
+    item.classList.add('active');
+
+    for (var i = 0; i < lists.length; i++) {
```

Comment:
`let` zamiast `var`, jeżeli stosowany jest ES6 to spójnie przez cały kod, bez mieszania ze 'starym' `var`.